lib/mzgl/util/Variable.h

```diff
@@ -73,6 +73,19 @@ class VariableWatcher : private Variable<T>::Listener {
 		variable.addListener(this);
 	}
 
+	VariableWatcher &operator=(const VariableWatcher &other) {
+		variable.removeListener(this);
+		variable = other.variable;
+		callback = other.callback;
+		variable.addListener(this);
+		return *this;
+	}
+
+	VariableWatcher(const VariableWatcher &other)
+		: variable(other.variable)
+		, callback(other.callback) {
+		variable.addListener(this);
+	}
```

Comment:
Does this need to call `removeListener` like the assigment operator?
Is it worth just doing `*this = other;`